README.adoc

```diff
@@ -119,6 +119,14 @@ can act as starting points or examples:
  provider such as https://stormpath.com/[Stormpath], Active Directory,
  https://developers.google.com/identity/[Google], http://openid.net/[OpenID], etc.
 
+ https://github.com/SMARTRACTECHNOLOGY/devkit-simple-service-example[Create Your Own Microservice]::
```

Comment:
I think this one is best placed here with the project and code it refers to.  This is not similar to the other DevKit guides that describe the DevKit and how to work with it.